### PR TITLE
Hichip qc v1

### PR DESCRIPTION
**Pull request by jared-dtg**
Version 1 of Hi-ChIP QC.

PR created on behalf of @dtg-jayghurye.

Adds simple Hi-Chip QC script to this repo.

### CODE REVIEW

**Comment by jared-dtg** [comment]
addresses #8

File: hichip_qc/hichip_qc.bash [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+ref=$1
+bam=$2
+peaks=$3
+prefix=$4
```
Comment (jared-dtg) [inline hichip_qc/hichip_qc.bash:6]: If not 4 arguments, print usage and exit.

**Comment by cpadillla1331** [comment]
Heads up in the install there is an error:EnvironmentFileNotFound: '/home/ubuntu/omni-c_qc/conda/environment.yaml' file not found


CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


Its either a typo in the code or in the file name because its looking for a file ending in ".yaml" but the file in the environment is ".yml". I changed the file end with ".yaml" and it worked

**Comment by cpadillla1331** [comment]
It also seems that you have to run

conda activate dovetail

Its unclear if I need to execute:
./create.sh

again, I did anyway just to be sure

**Comment by dtg-jayghurye** [comment]
How are you running that? You need to be in conda directory to run the create.sh script.

**Pull request merged**